tests/topotests/bgp_comm_list_match/test_bgp_comm_list_match.py

```diff
@@ -133,6 +133,70 @@ def _bgp_converge():
     assert result is None, "Failed to filter BGP UPDATES with community-list on R3"
 
 
+def test_bgp_comm_list_limit_match():
+    tgen = get_topogen()
+
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    router = tgen.gears["r3"]
+    router.vtysh_cmd(
+        """
+        configure terminal\n
```

Comment:
Please remove `\n` chars, newlines are handled out of the box.

Reply:
done